autobotAI_integrations/integrations/crowdstrike/__init__.py

```
@@ -126,3 +167,56 @@ def generate_steampipe_creds(self) -> SteampipeCreds:
+            }
+        )
+
+    def build_python_exec_combinations_hook(
+        self, payload_task: PayloadTask, client_definitions: List[SDKClient]
+    ) -> list:
+        falconpy = importlib.import_module("falconpy", package=None)
+        client_id = payload_task.creds.envs.get("FALCON_CLIENT_ID")
+        client_secret = payload_task.creds.envs.get("FALCON_CLIENT_SECRET")
+        client_cloud = payload_task.creds.envs.get("FALCON_CLOUD")
+
+        def _make_client(class_name: str):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_make_client`

(ANN202)